boilerplates/be-golang/files/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func main() {
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "Hello, world!")
+	})
+
+	http.ListenAndServe(":8080", nil)
```

Comment:
Nice to see that a valid app and Dockerfile is being built which will deploy without issues compared to https://github.com/opendevstack/ods-project-quickstarters/blob/master/boilerplates/be-docker-plain/files/docker/Dockerfile .

Reply:
@tbugfinder .. File a Bug - and provide a PR ;-) See you uptaking ODS?